wiki/Guides/Compressing_files/fr.md

```diff
@@ -0,0 +1,99 @@
+
+Si vous êtes sous Windows, commencez par [télécharger FFmpeg](https://ffmpeg.org/download.html) et ajoutez son répertoire à votre variable d'environnement `PATH`. Sous MacOS, vous pouvez également l'installer en utilisant [brew](https://brew.sh/). La plupart des distributions Linux ont déjà FFmpeg installé, mais il peut souvent être facilement installé par leurs gestionnaires de paquets respectifs.
+
+Ouvrez un terminal et collez la commande suivante, en modifiant les valeurs si nécessaire :
+
+```
+ffmpeg -i input -c:v libx264 -crf 20 -preset slower -an -sn -map_metadata -1 -map_chapters -1 -vf scale=-1:720 output.mp4
+```
+
+- `-i input` : Votre fichier source. Si le nom du fichier contient des espaces, mettez-le entre guillemets (`"`).
+- `-c:v libx264` : Indique que la vidéo doit être encodée à l'aide de l'encodeur x264, produisant une vidéo au format H.264.
+- `-crf 20` : La qualité de la compression, où des valeurs plus faibles donnent une meilleure qualité au détriment de fichiers plus volumineux et vice versa. La fourchette recommandée est d'environ 20-25.
+- `-preset slower` : Spécifiez un préréglage d'encodage, avec des valeurs recommandées allant de `ultrafast` à `veryslow`. Les préréglages plus lents permettent à l'encodeur de vous donner une qualité supérieure pour le même débit, ou un débit inférieur pour la même qualité. Vous trouverez plus d'informations sur les préréglages disponibles sur le [site officiel de FFmpeg](https://trac.ffmpeg.org/wiki/Encode/H.264#Preset).
+- `-an -sn` : Supprime l'audio et les sous-titres s'ils sont présents.
+- `-map_metadata -1 -map_chapters -1` : Supprime les métadonnées et les chapitres s'ils sont présents.
+- `-vf scale=-1:720` : Réduit l'échelle de la vidéo à une hauteur de 720 pixels. Le `-1` permet à FFmpeg de déterminer automatiquement la largeur de la nouvelle vidéo en fonction du ratio d'aspect de la source.
+- `output.mp4` : Votre fichier de sortie. Si le nom du fichier contient des espaces, mettez-le entre guillemets (`"`).
+
+## Audio
+
+Le débit binaire audio détermine en grande partie la taille du fichier audio. Vous pouvez utiliser [Audacity](https://www.audacityteam.org/) pour modifier le débit binaire de vos fichiers audio.
+
+Les [critères de classement](/wiki/Ranking_Criteria#audio) ont une règle qui stipule que tout ce qui a un débit moyen supérieur à 192 kbps n'est pas autorisé. En outre, tout ce qui est inférieur à 128 kbps est généralement considéré comme étant de faible qualité.
+
+### Utilisation de Audacity
+
+1. Importez le fichier audio dans Audacity.
+
+![Importer de l'audio dans Audacity](img/import-audacity-FR.png "Importer de l'audio dans Audacity")
```

Comment:
crop to the top-left corner

![image](https://user-images.githubusercontent.com/36758269/145276588-d0f02ec1-433f-403f-9ed6-645234d7ef8c.png)